lib/Dialect/TTNN/Utils/PassOverrides.cpp

```diff
@@ -6,6 +6,18 @@
 
 namespace mlir::tt::ttnn {
 
+const std::string OptionNames::optimizerPassEnabled = "enable-optimizer";
+const std::string OptionNames::overrideInputLayout = "insert-memreconfig";
+const std::string OptionNames::overrideOutputLayout = "override-output-layout";
```

Comment:
Nit: Might be cleaner just to have these set in the .h file?

Reply:
Done.